helper-npm-update

```diff
@@ -23,4 +23,4 @@ set -eu -o pipefail
 
 # HELPER COMMANDS
 
-npm update --dev --no-package-lock --no-save
+npm update --depth 9999 --dev --no-package-lock --no-save
```

Comment:
Looks good. I notice in the comments above someone has explained what the flags do, so it might be nice just to play along and add `--depth` in there too... :man_shrugging:

Reply:
Ok will do